src/components/Projects/ProjectCreate.tsx

```diff
@@ -0,0 +1,227 @@
+                                </FormLabel>
+                                <Input
+                                    variant="filled"
+                                    value={project_title}
+                                    placeholder="Title of your project"
+                                    onChange={(event) =>
+                                        setProjectTitle(event.target.value)
+                                    }
+                                />
+                                <FormHelperText>
+                                    Title of your project
+                                </FormHelperText>
+                            </Stack>
+                        </CardHeader>
+                        <CardBody>
+                            <Stack divider={<StackDivider />} spacing="4">
+                                <Image
+                                    src="https://images.unsplash.com/photo-1555041469-a586c61ea9bc?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80"
+                                    alt="Green double couch with wooden legs"
+                                    borderRadius="lg"
+                                />
+
+                                <Stat>
+                                    <StatLabel>
+                                        Target Amount (in USD)
+                                    </StatLabel>
+                                    <StatNumber>
+                                        <NumberInput
+                                            variant="filled"
+                                            min={0}
+                                            value={targeted_amount}
+                                            onChange={(value) => {
+                                                if (!value) {
+                                                    setTargetedAmount(0)
+                                                } else {
+                                                    setTargetedAmount(
+                                                        parseInt(value ?? '0')
+                                                    )
+                                                }
+                                            }}
+                                        >
+                                            <NumberInputField />
+                                        </NumberInput>
+                                    </StatNumber>
+                                </Stat>
+                                <Stat>
+                                    <StatLabel>Goal time</StatLabel>
+                                    <StatNumber>
+                                        <Input
+                                            type="date"
+                                            value={goal_time.toLocaleDateString(
+                                                'en-CA'
```

Comment:
why CA though?

Reply:
Is to wrangle the date object into the format that the browser allows e.g dd-mm-yyyy lol found it off some random stack overflow post

Reply:
So that I can pass a controlled (react) value to the input